public/js/app.Ctrl.js

```diff
@@ -0,0 +1,201 @@
+App.controller('App.Ctrl', function($scope, $timeout, AppService) {
+    const colorCode = {1:"green", 2: "red", 3: "yellow", 4: "blue"};
+    const speed = 500; // in milliseconds
+    var on = false;
+    var lock = false;
+    var strict = false;
+    var colorSequence = [];
+    var count = 0;
+    var round = 1;
+    var turnOffTimer;
+    var recursiveTimer;
+    var timer;
+    var sound = {1:{}, 2:{}, 3: {}, 4: {}};
+    var buzzer;
+    var win;
+
+    $scope.init = function() {
+        if(on === true) {
+            setUpSoundEffects();
+            reset(true);
+        }
+    };
+
+    $scope.toggleOnOff = function() {
+        if(on === true) {
+            on = false;
```

Comment:
I created a scope variable $scope.on and utilized the ng-class directive to change the css classes of the on/off buttons and strict button.